packages/avatar/package.json

```diff
@@ -24,7 +24,8 @@
   "peerDependencies": {
     "@hig/theme-context": "^3.0.1",
     "@hig/theme-data": "^2.22.0",
-    "react": "^15.4.1 || ^16.3.2"
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
```

Comment:
Hey @DevFoncy - I don't think we need react dom and the component level, just at the top level where we use render. You should be able to remove react-dom from each component package.

Reply:
Gotcha , you're right